espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/exceptions/MoveTargetOutOfBoundsException.java

```diff
@@ -2,14 +2,11 @@
 
 
 public class MoveTargetOutOfBoundsException extends AppiumException {
-    public MoveTargetOutOfBoundsException() {
-        super("The target for pointer interaction is not in the viewport and cannot be brought into that viewport");
-    }
 
-    public MoveTargetOutOfBoundsException(long targetX, long targetY) {
+    public MoveTargetOutOfBoundsException(long targetX, long targetY, long viewportWidth, long viewportHeight) {
```

Comment:
I'd rather pass the Rect here and show its toShortString Description, since the actual bounds might not only be a screen bounds, but also element bounds